index.js

```diff
@@ -1,4 +1,4 @@
 'use strict';
 const packageJson = require('package-json');
 
-module.exports = name => packageJson(name.toLowerCase()).then(data => data.version);
+module.exports = (name, version) => packageJson(name.toLowerCase(), version ? {version} : {}).then(data => data.version);
```

Comment:
`version ? {version} : {}` => if the `version` key exists (even if `undefined`), it is used in `package-json`, so if `version` is not passed to the function, we simply pass an empty object as `package-json` options